authui/src/authflowv2/base.css

```diff
@@ -63,7 +63,11 @@
     --color-neutral-700: #131315;
 
     /* background */
-    --color-bg-base: #ffffff;
+    --color-surface-primary: #ffffff;
```

Comment:
This should be defined as

```
--color-surface-primary: var(--color-white);
```

Reply:
@yyorkchan Move this below `color-white` and `color-black`, and address my previous comment.